controllers/replicationdestination_controller.go

```diff
@@ -297,7 +297,10 @@ func (r *rsyncDestReconciler) ensureJob(l logr.Logger) (bool, error) {
 		runAsUser := int64(0)
 		r.job.Spec.Template.Spec.Containers[0].SecurityContext = &corev1.SecurityContext{
 			Capabilities: &corev1.Capabilities{
-				Add: []corev1.Capability{"AUDIT_WRITE"},
+                                Add: []corev1.Capability{
+                                        "AUDIT_WRITE",
+                                        "SYS_CHROOT",
```

Comment:
This addition will also be needed for the replication source to handle cases where we have the destination pull from the source (meaning the source would be the one running sshd). That direction doesn't work yet, but... eventually.